internal/kuberesource/parts.go

```diff
@@ -25,10 +25,13 @@ func ContrastRuntimeClass(platform platforms.Platform) (*RuntimeClassConfig, err
 		return nil, fmt.Errorf("getting default runtime handler: %w", err)
 	}
 
+	// Consists of the default VM memory, 70MiB for the Kata shim and 100MiB for qemu overhead.
```

Comment:
Out of curiosity: Does each pod get its own shim?

Reply:
Yes, and because of the `sandbox_cgroup_only = true` change it's now being accounted towards pod memory, not in a Kata overhead group.